sleap_nn/architectures/heads.py

```diff
@@ -0,0 +1,41 @@
+"""Model head definitions for defining model output types."""
+
+from typing import Tuple, Union
+
+import torch
+from torch import nn
+
+
+class UNetOutputHead(nn.Module):
```

Comment:
Heads should not be backbone specific. We want to have model output type-specific heads that take a feature map as input, regardless of where it comes from.

See: https://github.com/talmolab/sleap/blob/develop/sleap/nn/heads.py